### PR TITLE
Leave a thank you note

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

File: site/docs/README.md
Comment (KnorpelSenf) [inline site/docs/README.md:90]: I'm not sure if we can simply update the copyright automatically lol

Comment (roj1512) [reply site/docs/README.md:90]: Isn’t that server-rendered?

File: site/docs/.vuepress/components/ThankYou.vue
Comment (KnorpelSenf) [inline site/docs/.vuepress/components/ThankYou.vue]: I still wanna find something better here than going through the list chronologically.

File: site/docs/README.md
Comment (KnorpelSenf) [reply site/docs/README.md:90]: It's both. The entire site is rehydrated client-side. Which is why I like islands so much. Remember the random quote above? It changes per visit.

File: site/docs/.vuepress/components/ThankYou.vue
Comment (roj1512) [reply site/docs/.vuepress/components/ThankYou.vue]: It doesn’t even work as is.

File: site/docs/README.md
Comment (roj1512) [reply site/docs/README.md:90]: Sad

File: site/docs/.vuepress/components/ThankYou.vue
Comment (KnorpelSenf) [reply site/docs/.vuepress/components/ThankYou.vue]: Your bracket placement is weird, too. You're doing mod on float now?

File: site/docs/README.md
Comment (KnorpelSenf) [reply site/docs/README.md]: Indeed

File: site/docs/.vuepress/components/ThankYou.vue
Comment (roj1512) [reply site/docs/.vuepress/components/ThankYou.vue]: It’s Prettier who did that. It also works in Node CLI.

Comment (KnorpelSenf) [reply site/docs/.vuepress/components/ThankYou.vue]: I was trying to say that you should have done `(Math.floor(Date.now() / 86400) % contributors.length)` not `Math.floor((Date.now() / 86400) % contributors.length)`

Comment (KnorpelSenf) [reply site/docs/.vuepress/components/ThankYou.vue]: But it does not matter anymore

**Comment by KnorpelSenf** [comment]
How can we translate this now? I think we somehow have to add a config option for the plugin akin to
```json
{
  "locales": {
    "/": { "label": "..." },
    "/es/": { "label": "..." },
}
```
but now we also need a way to interpolate a value in these strings …

**Comment by roj1512** [comment]
We will do it in `config.ts`.

File: site/docs/.vuepress/components/ThankYou.vue
Comment (roj1512) [reply site/docs/.vuepress/components/ThankYou.vue]: Gut. Danken fûr your footercontributorthankyounoterandomizationalgorithm.

Comment (KnorpelSenf) [reply site/docs/.vuepress/components/ThankYou.vue]: Ahahah rica dikim

Comment (roj1512) [reply site/docs/.vuepress/components/ThankYou.vue]: *Spas dikim

File: site/docs/.vuepress/components/ThankYou.vue
Comment (KnorpelSenf) [inline site/docs/.vuepress/components/ThankYou.vue:95]: eh
```suggestion
      {{ props.s[0] }}<span>{{ contributor.name }}</span>{{ props.s[2] }}
```

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
Please add an alt text to the image for a11y reasons. These changes reduce the lighthouse score for a11y by ~10 points.

**Review by KnorpelSenf — APPROVED** [review]
LGTM after translations are done.

File: site/docs/es/README.md
Comment (KnorpelSenf) [inline site/docs/es/README.md:87]: @habemuscode this is the only line that needs to be translated to Spanish

File: site/docs/id/README.md
Comment (KnorpelSenf) [inline site/docs/id/README.md:89]: @fadzikri @CikiMomogi this the only line that needs to be translated to Indonesian

File: site/docs/id/README.md
Comment (CikiMomogi) [inline site/docs/id/README.md:89]: We can shorten it to 'Terima kasih Someone telah menjadi kontributor GrammY.' It sounds more natural to me.



```suggestion
  <ThankYou :s="['Terima kasih ', 'Someone', ' telah menjadi kontributor grammY.']" />
```

**Review by CikiMomogi — CHANGES_REQUESTED** [review]

**Comment by CikiMomogi** [comment]
The contributor name is not changing when the page is refreshed.

File: site/docs/zh/README.md
Comment (CikiMomogi) [inline site/docs/zh/README.md:90]:
```suggestion
<div style="font-size: 0.75rem; display: flex; justify-content: center;">
```

File: site/docs/id/README.md
Comment (CikiMomogi) [inline site/docs/id/README.md:92]:
```suggestion
<div style="font-size: 0.75rem; display: flex; justify-content: center;">
```

File: site/docs/README.md
Comment (CikiMomogi) [inline site/docs/README.md:90]:
```suggestion
<div style="font-size: 0.75rem; display: flex; justify-content: center;">
```

File: site/docs/es/README.md
Comment (CikiMomogi) [inline site/docs/es/README.md:90]:
```suggestion
<div style="font-size: 0.75rem;  display: flex; justify-content: center;">
```

**Review by CikiMomogi — COMMENTED** [review]
Center the div?

File: site/docs/id/README.md
Comment (fadzikri) [reply site/docs/id/README.md:89]: Ok.

**Comment by fadzikri** [comment]
@CikiMomogi di you delete space between someone 😅.

**Comment by CikiMomogi** [comment]
Review commit [62e2430](https://github.com/grammyjs/website/pull/689/commits/62e2430f44c24fb40ed201e6fec2bb8373bb24fe), please.

**Comment by CikiMomogi** [comment]
This commit adds a function that displays a single contributor per day based on a fixed index instead of random, giving each contributor equal exposure. It has the same behavior as the previous commit.

**Comment by CikiMomogi** [comment]
I see, so you want the surprise element. Then I will revert it.

**Comment by KnorpelSenf** [comment]
Also, the pseudo-random function will over time also even out chances for everyone to appear. We've tested this and made sure this is actually the case. Moreover, note that the length of the list again changes the distribution, so for every new contributor there will be new chances.

**Comment by KnorpelSenf** [comment]
> I see, so you want the surprise element. Then I will revert it.

Thanks!

**Review by roj1512 — COMMENTED** [review]
- Gracias, @habemuscode, por ser colaborador de grammY.
- Terima kasih @CikiMomogi telah menjadi kontributor grammY.
- Terima kasih @fadzikri telah menjadi kontributor grammY.
- 感谢 “ “ 成为 grammY 的贡献者。

**Comment by KnorpelSenf** [comment]
While most people spend time on the website to learn about grammY, @roj1512 spends time on the website to learn the world's languages

**Comment by roj1512** [comment]
I will especially be happy when there are the Cyrillic languages in our website, I am much more into them :)

**Comment by dcdunkan** [comment]
Θ Thank you roj1512 for being a contributor to grammY.

**Comment by Nazar-Ant** [comment]
Could you tell me, these are not the same critical changes as before? If I just do a merge of the `main` and `uk` branches and then push some changes, won't it break the translation branches? Do I need to merge the `uk` branch into the translation branch afterwards?

**Comment by KnorpelSenf** [comment]
Your translation branch `uk` is long-lived and you should regularly merge `main` into it. There technically shouldn't be any merge conflicts because most of the translation process happens in a new subdirectory.

The branches for each page are short-lived, they usually only exist for a few days or less. This means that `uk` doesn't evolve much further in the lifetime of such a branch, so there shouldn't be any conflicts, either. Also, new translations happen inside new files, which further reduces the risk of merge conflicts.

The only real thing to pay attention to is that `main` receives new changes all the time. This means that there are updates to the documentation in English, Spanish, Indonesian, and Chinese. Whenever such a PQ is merged into `main`, the changes need to be synced on `uk`, too (assuming that the affected pages are translated already). It may make sense to create separate pull requests for syncing up with `main`.

**Pull request merged**
